lib/node_modules/@stdlib/blas/ext/base/wasm/snansumors/README.md

```diff
@@ -0,0 +1,291 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2025 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# snansumors
+
+> Calculate the sum of single-precision floating-point strided array elements, ignoring `NaN` values and using ordinary recursive summation.
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var snansumors = require( '@stdlib/blas/ext/base/wasm/snansumors' );
+```
+
+#### snansumors.main( N, x, strideX )
+
+Computes the sum of single-precision floating-point strided array elements, ignoring `NaN` values and using ordinary recursive summation.
+
+```javascript
+var Float32Array = require( '@stdlib/array/float32' );
+
+var x = new Float32Array( [ 1.0, -2.0, NaN, 2.0 ] );
+
+var v = snansumors.main( x.length, x, 1 );
+// returns 1.0
+```
+
+The function has the following parameters:
+
+-   **N**: number of indexed elements.
+-   **x**: input [`Float32Array`][@stdlib/array/float32].
+-   **strideX**: stride length for `x`.
+
+The `N` and stride parameters determine which elements in the strided arrays are accessed at runtime. For example, to compute the sum of every other element in `x`,
+
+```javascript
+var Float32Array = require( '@stdlib/array/float32' );
+
+var x = new Float32Array( [ 1.0, 2.0, NaN, -7.0, NaN, 3.0, 4.0, 2.0 ] );
+
+var sum = snansumors.main( 4, x, 2 );
+// returns 5.0
+```
+
+Note that indexing is relative to the first index. To introduce an offset, use [`typed array`][mdn-typed-array] views.
+
+<!-- eslint-disable stdlib/capitalized-comments -->
+
+```javascript
+var Float32Array = require( '@stdlib/array/float32' );
+
+var x0 = new Float32Array( [ 2.0, 1.0, NaN, -2.0, -2.0, 2.0, 3.0, 4.0 ] );
+var x1 = new Float32Array( x0.buffer, x0.BYTES_PER_ELEMENT*1 ); // start at 2nd element
+
+var v = snansumors.main( 4, x1, 2 );
+// returns 5.0
+```
+
+#### snansumors.ndarray( N, x, strideX, offsetX )
+
+Computes the sum of single-precision floating-point strided array elements, ignoring `NaN` values and using ordinary recursive summation indexing semantics.
```

Comment:
@0PrashantYadav0 Are you using generative AI on these PRs? If you are, I would advise against doing so. This description is incorrect and should be fixed wherever you've done something similar.